samples/nrf9160/modem_shell/README.rst

```diff
@@ -339,6 +342,10 @@ Set the PPP network interface down::
 
    ppp down
 
+Set the custom configuration for ppp uart::
+
+   ppp uartconf <options>
```

Comment:
Use real options here as an example so remove `<options>` and add something that could work in correct setup. Similarly, sms phone number in an example is +987654321.

Reply:
Done.